devito/interfaces.py

```diff
@@ -18,8 +56,23 @@ def set_initializer(self, lambda_initializer):
         assert(callable(lambda_initializer))
         self.initializer = lambda_initializer
 
+    # function to generate a hased file name used for memmap file
+    def _generate_memmap_filenmae(self):
```

Comment:
Hi, thank you very much. I can implement the user input, but I believe that we cant have defaults of the form "/tmp/devito-memmap/data_%s" % self.name, because there can be multiple "u" or "v" generated, this will overwritten previous data, therefore we are using a hash. Regarding the user entered name, I think that a prefix might be more appropriate because it will be quite a hassle for the user to input a filename every time a file is to be created. What do you think?